ROADMAP.md

```diff
@@ -74,12 +74,13 @@ Using `gfx-rs` for backend abstraction allows us to provide compatibility with m
 **Next steps**
  * Raw input support (winit?)
  * Gamepad support (winit?)
- * Input Mapping
+ * Input Mapping: Map input events (e.g key press) to abstract action events (`Jump`, `Forward`, ..)
```

Comment:
This is a good point! 👍 We obviously need to improve the ergonomics of this in the long term by building this functionality into the engine proper. From a UX perspective, matching over simple strings comes to mind, or perhaps an `Action ` tuple struct which contains the string and also the keycode which triggered the action.